test/k8s/common.go

```diff
@@ -0,0 +1,17 @@
+package k8s
```

Comment:
Looking at the import path made me thinking, what would happen when you want to share code between multiple test platforms.
will `local` also need to import `test/k8s`?

perhaps we can nest it differently, such as
`test/common/k8s/ingress.go`

and if, for an instance, a functionality from `ingress.go` becomes relevant to other platform, then it would be simply moved to `test/common/ingress.go`

food for thought?

Reply:
sounds good